backend/db/seeds.rb

```diff
@@ -151,6 +151,57 @@
   })
 end
 
+if DataPackage.where(package: "public_schools", version: "november_2019").first.nil?
+  DataPackage.create({
+    name: "November 2019",
+    version: "november_2019",
+    package: "public_schools",
+    release_date: Date.parse('2019-11-01'),
+    schemas: {
+      "doe_school_subdistricts": { table: "2017" },
+      "ceqr_school_buildings": {
+        table: 2019,
+        sources: [
+          {name: "lcgms", version: '2019-12-19', minYear: 2018, maxYear: 2019},
+          {name: "bluebook", minYear: 2017, maxYear: 2018}
+        ]
+      },
+      "doe_school_zones_ps": { table: "2019" },
+      "doe_school_zones_is": { table: "2019" },
+      "doe_school_zones_hs": { table: "2019" },
+      "sca_enrollment_pct_by_sd": { table: "2019" },
+      "sca_housing_pipeline_by_boro": {
+        minYear: 2018, # TODO: Update
+        maxYear: 2027, # TODO: Update
+        table: "2019"
+      },
+      "sca_housing_pipeline_by_sd": {
+        minYear: 2018, # TODO: Update
+        maxYear: 2027, # TODO: Update
+        table: "2019"
+      },
+      "sca_enrollment_projections_by_boro": {
+        minYear: 2017, # TODO: Update
+        maxYear: 2027, # TODO: Update
+        table: "2019"
+      },
+      "sca_enrollment_projections_by_sd": {
+        minYear: 2017, # TODO: Update 
+        maxYear: 2027, # TODO: Update
+        table: "2019"
+      },
+      "doe_significant_utilization_changes": {
+        table: "042019",
+        version: "2019-04-01" # TODO: Update, determine what "version" this is
+      },
+      "sca_capital_projects": {
+        table: "2019",
+        version: "2018-12-04" # TODO: Update, determine what "version" this is
+      }
```

Comment:
it can't find "sca_capacity_projects" — this says capital_projects. Is there a difference? @bfreeds